test/integration/testImage/yaml/ds-junit-passing-raw.yaml

```diff
@@ -0,0 +1,18 @@
+sonobuoy-config:
```

Comment:
Is this a leftover file? It's not managed by the `generate.sh` script.

Reply:
Removed; thanks. Old file that should have gotten cleaned up.